src/errors/checkXcmTxInputs.ts

```diff
@@ -504,7 +512,7 @@ export const checkParaAssets = async (
 		);
 
 		if (!multiLocationIsInRegistry) {
-			// TODO: create AssetHub ApiPromise to query chain state for foreign assets
+			// TODO: remove foreignAssets check from ParaToSystem
```

Comment:
Based on what we last spoke about ,right now we are only aiming to support the native tokens for paras and making the UX the same as it is for DOT and relay assets. So we could keep the registry check for sure but Im not sure how valid it will be with paratosystem logic when we begin to support other para native assets.